scripts/movement_state.gd

```diff
@@ -32,15 +39,6 @@ func _init():
 	add_transition("Dash", "Moving")
 	add_transition("Dash", "Idle")
 
-func _ready():
-	coyote_timer = Timer.new()
```

Comment:
If we are using the game tee method this shouldn't be here anymore

Reply:
its not one way or the other. this is simpler to merge to do since you did additional fixes :^)

Reply:
fixed in https://github.com/Dragonlinae/Papereaven/pull/17/commits/27d61af496f1b0a9877e87eb95ce430a64f4f3f1